jicoco-config/src/main/kotlin/org/jitsi/config/JitsiConfig.kt

```diff
@@ -31,10 +32,48 @@ class JitsiConfig {
     companion object {
         val logger = LoggerImpl(JitsiConfig::class.simpleName)
 
+        var typesafeConfigLoader: () -> Config = {
```

Comment:
I intentionally made this a var, thinking it may be a nice hook to be able to override.  We _could_ put this logic in JVB itself and leave this as some default logic, but it felt like we'd want this behavior everywhere.

Reply:
To be overridable wouldn't it need to be an `open var`?

Reply:
Sorry, by "override" I guess I meant "overwrite", `JitsiConfig` isn't open.